src/SignupFields.js

```diff
@@ -32,49 +32,44 @@ const SignupFields = ({ form, fields }) => {
     USERNAME_AND_EMAIL,
     USERNAME_AND_OPTIONAL_EMAIL,
   } = PasswordSignupFields;
-  let jsx;
   switch (AccountsClient.options().passwordSignupFields) {
     case EMAIL_ONLY:
-      jsx = (
+      return (
         <div>
           <AccountsClient.ui.EmailField {...emailProps({ form, fields })} />
           <AccountsClient.ui.PasswordField {...passwordProps({ form, fields })} />
           <AccountsClient.ui.PasswordConfirmField {...passwordConfirmProps({ form, fields })} />
         </div>);
-      break;
     case USERNAME_ONLY:
-      jsx = (
+      return (
         <div>
           <AccountsClient.ui.UsernameField {...usernameProps({ form, fields })} />
           <AccountsClient.ui.PasswordField {...passwordProps({ form, fields })} />
           <AccountsClient.ui.PasswordConfirmField {...passwordConfirmProps({ form, fields })} />
         </div>
       );
-      break;
     case USERNAME_AND_EMAIL:
-      jsx = (
+      return (
         <div>
           <AccountsClient.ui.UsernameField {...usernameProps({ form, fields })} />
           <AccountsClient.ui.EmailField {...emailProps({ form, fields })} />
           <AccountsClient.ui.PasswordField {...passwordProps({ form, fields })} />
           <AccountsClient.ui.PasswordConfirmField {...passwordConfirmProps({ form, fields })} />
         </div>
       );
-      break;
     case USERNAME_AND_OPTIONAL_EMAIL:
-      jsx = (
+      return (
         <div>
           <AccountsClient.ui.UsernameField {...usernameProps({ form, fields })} />
           <AccountsClient.ui.EmailOptionalField {...emailProps({ form, fields })} />
           <AccountsClient.ui.PasswordField {...passwordProps({ form, fields })} />
           <AccountsClient.ui.PasswordConfirmField {...passwordConfirmProps({ form, fields })} />
         </div>
       );
-      break;
     default:
       break;
   }
-  return jsx;
+  return null;
```

Comment:
Lol really this should go under the `default` case. At the end of the day I don't care.